src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Android.cs

```diff
@@ -0,0 +1,390 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System
+{
+    public sealed partial class TimeZoneInfo
+    {
+        private const string _timeZoneFileName = "tzdata";
+
+        private static AndroidTzData? s_tzData;
+        private static readonly object s_tzDataLock = new object();
+
+        private static AndroidTzData s_androidTzDataInstance
```

Comment:
This is a property, not a field, so it should be PascalCased. `s_` is only for static fields.

Reply:
Thanks!